server/.env

```diff
@@ -1,4 +1,4 @@
 PORT=2999
-CLIENT_URL=http://127.0.0.1:5173/
+CLIENT_URL=http://127.0.0.1:5173
 STREAM_API_KEY=hvwxgw75ahm7
 STREAM_PRIVATE_API_KEY=3tne74rq3dzv5htptmccnm8kcqzcvj5yg2ymxv3cunsj5ayx2c7e5nhcvmgzf3cy
```

Comment:
lets remove this from `.env` file

exposing API key is not good , we will start getting emails from github and other bots most probably if we merge this in main

- the person who is using this app locally must update this variable in their `env` file manually every time they are testing it, 
- in remote deployed version of this app, the variable will be available at runtime , injected from github cd pipeline (deployment pipeline) from github secrets

Reply:
okay